src/app/header/header.component.css

```diff
@@ -183,13 +183,13 @@ hr {
     color:#858585;
 } */
 
-.dark-mode-theme {
+.dark-mode-informative-text {
     padding: 1rem;
     padding-top: 0.3rem;
     font-size: 0.8rem;
     color: rgb(180, 180, 180);
 }
```

Comment:
Add a new line below, same for line 136 in `src/app/settings/settings.component.css.`